guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>The trained model could now be saved using the <code><em>model.save(name_or_path)</em></code> command, which will save the model weights, creates an Esri Model Definition and Deep Learning Package zip <s>for deployment to Image Server or ArcGIS Pro. </s>For example </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>